### DISCUSSION TITLE
what is the library to use for react api handling

### DISCUSSION THREAD

**Discussion by jsMasterDND** [community/community#123792] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

i need to know what is the library to use for react api handling

**Comment by IroshanBrian** [comment]
use axios or js fetch method but i recommend axios

**Comment by jsMasterDND** [comment]
axios

**Comment by ebndev** [comment]
Hi @jsMasterDND & @IroshanBrian,

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
